action.yml

```diff
@@ -63,21 +63,7 @@ runs:
       if: inputs.annotate == 'true'
     - run: |
         # The Julia command that will be executed
-        julia_cmd=( julia --color=yes --depwarn=${{ inputs.depwarn }} --inline=${{ inputs.inline }} --project=${{ inputs.project }} -e '
-        import Pkg
-        include(joinpath(ENV["GITHUB_ACTION_PATH"], "kwargs.jl"))
-        kwargs = Kwargs.kwargs(;coverage = :(${{ inputs.coverage }}),force_latest_compatible_version = :(${{ inputs.force_latest_compatible_version }}), julia_args = ["--check-bounds=${{ inputs.check_bounds }}"])
-
-        if ${{ inputs.annotate }} && VERSION > v"1.8pre"
-          push!(LOAD_PATH, "@tests-logger-env") # access dependencies
-          using GitHubActions, Logging
-          global_logger(GitHubActionsLogger())
-          include(joinpath(ENV["GITHUB_ACTION_PATH"], "test_wrapper.jl"))
-          pop!(LOAD_PATH)
-          TestWrapper.test(; kwargs...)        
-        else
-          Pkg.test(; kwargs...)        
-        end' )
+        julia_cmd=( julia --color=yes --depwarn=${{ inputs.depwarn }} --inline=${{ inputs.inline }} --project=${{ inputs.project }} -e 'include(joinpath(ENV["GITHUB_ACTION_PATH"], "test_harness.jl"))' )
```

Comment:
I still have `-e` because I don't know how to do the joinpath and env variables in bash

Reply:
I think `"$GITHUB_ACTION_PATH/test_harness.jl"` would do the trick but this is more approachable from a Julia PoV so it's probably better.